src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Mashine {

```

Comment:
The class name `Mashine` is likely a typo. It should be `Machine` according to the task description. Please correct this to ensure consistency and functionality.